akka-actor-typed/src/main/resources/reference.conf

```diff
@@ -17,6 +17,12 @@ akka.actor.typed {
 
   # Receptionist is started eagerly to allow clustered receptionist to gather remote registrations early on.
   library-extensions += "akka.actor.typed.receptionist.Receptionist"
+
+  # While an actor is restarted (waiting for backoff to expire and children to stop)
+  # incoming messages and signals are stashed, and delivered later to the newly restarted
+  # behavior. This property defines the capacity in number of messages of the stash
+  # buffer. If the capacity is exceed then additional incoming messages are dropped.
+  restart-stash-capacity = 1000
```

Comment:
Do we deliver the first 1000 or the last 1000 messages?

Reply:
the first: " If the capacity is exceed then additional incoming messages are dropped."